releasenotes/notes/add-windows-permissions-abfb73d19318641a.yaml

```
@@ -0,0 +1,14 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Added parity to Linux-based permissions.log Flare file
+    on Window to include permissions of Datadog agent
+    configuration directory (by default C:\ProgramData\Datadog)
+    and Agent executable files.
```

We should update this now that we fully mimic the linux behavior, no ?

What about:
```suggestion
    Added parity to Unix-based ``permissions.log`` Flare file on Windows. ``permissions.log`` file list the original
    rights/ACL of the files copied into a Agent flare. This will ease troubleshooting permissions issues.
```

---

It is pretty good suggestion. Actually very well writtent. I am indeed "stealing" it. Thank you.